db/range_tombstone_fragmenter.h

```diff
@@ -218,8 +218,7 @@ class FragmentedRangeTombstoneIterator : public InternalIterator {
   }
 
   ParsedInternalKey parsed_start_key() const {
-    return ParsedInternalKey(pos_->start_key, kMaxSequenceNumber,
-                             kTypeRangeDeletion);
+    return ParsedInternalKey(pos_->start_key, seq(), kTypeRangeDeletion);
```

Comment:
This may cause inconsistency between the timestamp portion of start_key (which is always kMaxTs) and sequence number. I plan to fix it in a separate PR, and store one start_key per timestamp so that parsed_start_key have timestamp and sequence number in sync.